extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -336,7 +338,10 @@ async function runRemoteQueriesApiRequest(
     );
     const { popupMessage, logMessage } = parseResponse(owner, repo, response.data);
     void showAndLogInformationMessage(popupMessage, { fullMessage: logMessage });
-    return response.data.workflow_run_id;
+    return {
```

Comment:
Feel free to ignore this, but just a thought: Here we could also return the whole response (`response.data`) and let the caller select the fields it needs. Then your return signature (`Promise<void | { workflowRunId: number, numRepositoriesQueried: number }>`) would be a bit more tidy too. E.g. `Promise<void | RunQueryResponse>` or something like that.

Reply:
Ah yes, good point - we already have the type `QueriesResponse`, so I've changed the function to return the whole thing 👍🏽